src/modules/select/index.ts

```diff
@@ -232,8 +238,8 @@ function getModule(name: string): Promise<{
       return import('./wallets/mathwallet')
     case '1inch':
       return import('./wallets/1inch')
-    case 'blankwallet':
-      return import('./wallets/blankwallet')
+    case 'blockwallet':
```

Comment:
```suggestion
    case 'blockwallet':
    case 'blankwallet':
```

To make this a non-breaking change, we'll need keep `'blankwallet'` as an option.